### PR TITLE
Edit section 2.5

### PR DESCRIPTION
**Pull request by dmarti**
Minor edits trying for clarity

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/364.html" title="Last updated on Oct 31, 2023, 11:17 PM UTC (30dc770)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/364/5853646...30dc770.html" title="Last updated on Oct 31, 2023, 11:17 PM UTC (30dc770)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1381,20 +1381,20 @@
 
 This right includes both being able to review what information has been collected or inferred about
 oneself and being able to discover what [=actors=] have collected information about oneself. As a
-result, databases cannot be kept secret and data collected about [=people=] needs to be meaningfully
+result, databases containing information about [=people=] cannot be kept secret and data collected about people needs to be meaningfully
```
Comment (jyasskin) [inline index.html:1384]: A minor tweak on top of yours:
```suggestion
result, databases containing information about [=people=] cannot be kept secret, and data collected about people needs to be meaningfully
```

File: index.html [hunk 2]
```diff
@@ -1417,11 +1417,11 @@
   data-lt="right to object|right to withdraw consent|right to restrict use">right to object,
 withdraw consent, and restrict use</dfn> of data about oneself.
 
-[=People=] may change their decisions about consent or may object to subsequent uses of data about
-themselves. Retaining rights requires ongoing control, not just at the time of collection.
+[=People=] may change their decisions about [=consent=] or may object to subsequent uses of data about
+themselves. Data rights mean that a person has ongoing control, not just a choice at the time of collection.
```
Comment (jyasskin) [inline index.html:1421]:
```suggestion
themselves. Data rights mean that a person needs to have ongoing control, not just a choice at the time of collection.
```

**Review by jyasskin — APPROVED** [review]
2 minor suggestions:

**Review by npdoty — APPROVED** [review]

**Review by darobin — APPROVED** [review]

**Review by torgo — APPROVED** [review]
lgtm

**Pull request merged**